service/ui.go

```diff
@@ -231,6 +231,9 @@ func (uis *UIServer) NewRouter() (*mux.Router, error) {
 	r.HandleFunc("/project/{project_id}", uis.loadCtx(uis.requireAdmin(uis.addProject))).Methods("PUT")
 	r.HandleFunc("/project/{project_id}/repo_revision", uis.loadCtx(uis.requireAdmin(uis.setRevision))).Methods("PUT")
 
+	// Admin routes
+	r.HandleFunc("/admin", requireLogin(uis.loadCtx(uis.adminSettings))).Methods("GET")
```

Comment:
I'm forgetting the scope/design--is the admin page accessible to all logged-in users or just superusers?

Reply:
Just superusers. Trying to access the page while not logged in or logged in as a non-superuser will show you this:
![image](https://user-images.githubusercontent.com/29384396/29892463-9cef61f4-8d9c-11e7-8342-ed50ed4bc1a5.png)

Reply:
Shouldn't this be `uis.requireSuperUser` instead of `requireLogin`?

Reply:
I think requireLogin is better. Making the url check for a logged in superuser means that someone who is logged in as a regular user will keep getting a login prompt until they log in as a superuser which will be confusing. The current implementation will show this page